src/NBInclude.jl

```diff
@@ -57,7 +57,9 @@ it is assigned a number `+N` for the `N`-th nonempty cell.  If `renumber`
 is set to `true`, then the cell numbers saved in the notebook are ignored
 and each cell is assigned a consecutive number `N`.
 """
-function nbinclude(path::AbstractString; renumber::Bool=false)
+function nbinclude(path::AbstractString; renumber::Bool=false,
+                                         counters = 1:typemax(Int),
+                                         regexp = r"")
```

Comment:
`regex::Regex` (not `regexp`, since "regex" seems to be the canonical abbreviation in Julia)